rapid7_insightvm_cloud/plugin.spec.yaml

```diff
@@ -0,0 +1,1202 @@
+      title: Port
+      type: integer
+      required: false
+    product:
+      description: Product running the service
+      title: Product
+      type: string
+      required: false
+    protocol:
+      description: Protocol of the service
+      title: Protocol
+      type: string
+      required: false
+    userGroups:
+      description: User groups
+      title: User Groups
+      type: '[]userGroup'
+      required: false
+    users:
+      description: Users
+      title: Users
+      type: '[]user'
+      required: false
+    vendor:
+      description: Vendor of the service
+      title: Vendor
+      type: string
+      required: false
+    version:
+      description: Version of the service
+      title: Version
+      type: string
+      required: false
+    webApplications:
+      description: Web applications found on the service
+      title: Web Applications
+      type: '[]webApplication'
+      required: false
+  vulnerabilities:
+    critical:
+      description: Number of critical vulnerabilities
+      title: Critical
+      type: integer
+      required: false
+    exploits:
+      description: Number of distinct exploits that can exploit any of the vulnerabilities
+        on the asset
+      title: Exploits
+      type: integer
+      required: false
+    malwareKits:
+      description: Number of distinct malware kits that vulnerabilities on the asset
+        are susceptible to
+      title: Malware Kits
+      type: integer
+      required: false
+    moderate:
+      description: Number of moderate vulnerabilities
+      title: Moderate
+      type: integer
+      required: false
+    severe:
+      description: Number of severe vulnerabilities
+      title: Severe
+      type: integer
+      required: false
+    total:
+      description: Total number of vulnerabilities
+      title: Total
+      type: integer
+      required: false
+  software:
+    configurations:
+      description: Configurations
+      title: Configurations
+      type: '[]configuration'
+      required: false
+    cpe:
+      description: CPE
+      title: CPE
+      type: cpe
+      required: false
+    description:
+      description: Description of the software
+      title: Description
+      type: string
+      required: false
+    family:
+      description: Family of the software
+      title: Family
+      type: string
+      required: false
+    id:
+      description: ID
+      title: ID
+      type: integer
+      required: false
+    product:
+      description: Product of the software
+      title: Product
+      type: string
+      required: false
+    type:
+      description: Type of the software
+      title: Type
+      type: string
+      required: false
+    vendor:
+      description: Vendor of the software
+      title: Vendor
+      type: string
+      required: false
+    version:
+      description: Version of the software
+      title: Version
+      type: string
+      required: false
+  asset:
```

Comment:
Mix of camelcase for types in asset? can we make sure these align with data returned from api?